platform/binance/model.go

```diff
@@ -174,3 +188,16 @@ func convertValue(value interface{}) (float64, bool) {
 	}
 	return result, true
 }
+
+// isNotZeroValue check that string value is not 0
+func isNotZeroValue(value string) bool {
```

Comment:
I would recommend to remove this function

Reply:
done